src/drive/web/modules/viewer/styles.styl

```diff
@@ -0,0 +1,57 @@
+@require 'settings/breakpoints.styl'
+@require 'settings/palette.styl'
+
+.pho-viewer-noviewer-cta
+    position         relative
+    border-radius    .5rem
+    background-color rgba(255, 255, 255, 0.05)
```

Comment:
Do we need a css var for that?

Reply:
I haven't actually looked at that in this PR, but mayyybe? If the color scheme is inverted then this would'nt work too well.

cc @GoOz what do you think?

Reply:
Well we don't have variables for colors with alpha transparency except the one for the layer of modals.
There's no easy way to do this with CSS custom properties sadly and I'm not sure adding in UI new color variables for every transparency need is a good solution. But I don't really have a good solution either… :|